src/NServiceBus.Core.Tests/NServiceBus.Core.Tests.csproj

```diff
@@ -297,21 +294,27 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="TestDlls\dotNet.dll">
+    <None Include="TestDlls\AssemblyWithNoDirectReference.dll">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="TestDlls\libzmq-v120-mt-3_2_3.dll">
+    </None>
+    <None Include="TestDlls\AssemblyWithReference.dll">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="TestDlls\some_random.dll">
+    </None>
+    <None Include="TestDlls\dotNet.dll">
```

Comment:
Could we somehow prefix the assemblies with the test they belong? When I come back in a few months and see dotNet.dll or some_random.dll I might not remember why they are here and what they are doing